Add community calendar, GeoJupyter announcement blog post, update about page (#6)
geojupyter/geojupyter.org: blog/20250108-introducing-geojupyter/index.md

Author: Really trying to keep this a quick read while still hitting the key points.

---
title: "Introducing GeoJupyter"
date: "2025-01-08"
description: |
  We believe that geospatial data analysis and exploration should be free, open source,
  and accessible for everyone!
author:
  - name: "Matt Fisher"
    orcid: "0000-0003-3260-5445"
categories:
  - "Announcements"
---

## :jigsaw: Need

**We believe that geospatial data analysis and exploration should be free, open source,
and accessible for everyone!**

> Currently, a divide exists between two worlds: The desktop GIS world, in which QGIS and

Reviewer: I would add some language that acknowledges this is a first approximation/simplification, b/c we know that all modern desktop GIS GUIs offer scripting options, and that many code-centric tools also integrate with some GUI elements (e.g. Qiusheng's [WhiteBox](https://github.com/opengeos/whiteboxgui) and I'm sure others).

But we're simply framing the basics of the picture, so we'll go with this simplified version, we're not trying to get into those finer points - our goal is to leverage the architecture of Jupyter to support better modern worfklows.

Author: :100: good call! Thinking more about this.

Author: 873ce1f

ArcGIS provide intuitive visualization-first interfaces to exploring data; and the
programming-based GIS world which provides cutting-edge editing, reproducibility, and
automated publishing experiences.

Between those two worlds, users of programming-based workflows face the daunting problem
of a blank page and steep learning curves, while non-coders are missing out on the
power of programming-based workflows to enable flexible alterations, version control,

> and low-friction reproducibility and publishing.

Reviewer: I'd update to "low-friction modular reuse, reproducibility and publishing."

Author: >modular reuse

Ooo, love this!

Author: 01f1612

  and accessible for everyone!
author:
  - name: "Matt Fisher"
    orcid: "0000-0003-3260-5445"
categories:
  - "Announcements"
---

## :jigsaw: Need

**We believe that geospatial data analysis and exploration should be free, open source,
and accessible for everyone!**

Currently, a divide exists between two worlds: The desktop GIS world, in which QGIS and
ArcGIS provide intuitive visualization-first interfaces to exploring data; and the
programming-based GIS world which provides cutting-edge editing, reproducibility, and
automated publishing experiences.

Between those two worlds, users of programming-based workflows face the daunting problem
of a blank page and steep learning curves, while non-coders are missing out on the
power of programming-based workflows to enable flexible alterations, version control,
and low-friction reproducibility and publishing.

## :telescope: Vision

![An animation from the 1990s childrens' TV program "Captain Planet" depicting the planeteers combining their powers to summon... Captain Planet!](https://media1.tenor.com/m/hWtovfyLjNAAAAAd/captain-planet-let-our-powers-combine.gif)

<div class="quote">
{{< include /elevator-pitch.md >}}
</div>

> GeoJupyter will comprise a new generation of tools and enhancements to existing tools
> supporting exploring data about our planet. This includes tasks traditionally done

Reviewer 2: Sentence read a little funny, maybe something like this?

```suggestion
GeoJupyter comprises of a new generation of tools and enhancements to existing
libraries supporting data exploration of our planet. This includes tasks traditionally done
```

Author: Thanks!

I think "comprises of" is not correct; would you prefer "consists of" or "comprises"?

> supporting data exploration of our planet

Feels like not quite there yet; what about  "supporting exploration of data about our planet"?

Reviewer 2: > I think "comprises of" is not correct; would you prefer "consists of" or "comprises"?

Let's go with 'consists of'. I think you're correct that it should be just 'comprises' if I'm reading https://dictionary.cambridge.org/grammar/british-grammar/consist-comprise-or-compose correctly, but it reads strangely (at least to me) if you say 'GeoJupyter comprises a new generation of ...'.

> Feels like not quite there yet; what about "supporting exploration of data about our planet"?

Yep, that works too.

Author: >it reads strangely (at least to me) if you say 'GeoJupyter comprises a new generation of ...'

This is super useful feedback, thanks! That's a good enough reason to change it :)

Author: 0d118ce
22d84ba

---
title: "Introducing GeoJupyter"
date: "2025-01-08"
description: |
  We believe that geospatial data analysis and exploration should be free, open source,
  and accessible for everyone!
author:
  - name: "Matt Fisher"
    orcid: "0000-0003-3260-5445"
categories:
  - "Announcements"
---

## :jigsaw: Need

**We believe that geospatial data analysis and exploration should be free, open source,
and accessible for everyone!**

Currently, a divide exists between two worlds: The desktop GIS world, in which QGIS and
ArcGIS provide intuitive visualization-first interfaces to exploring data; and the
programming-based GIS world which provides cutting-edge editing, reproducibility, and
publishing experiences.

Between those two worlds, users of code-based workflows face the daunting problem of a
blank page and steep learning curves, while non-programmers are missing out on the
power of code-based workflows to enable flexible alterations, version control, and

> low-friction reproducibility and publishing.

Author: I feel this is a bit awkward, but having trouble putting my finger on what exactly is bothering me,

> ## :telescope: Vision

Author: :crystal_ball: ? :)

Reviewer 2: For the vision, maybe we can use the term 'multi-player mode' to differentiate JupyterGIS from the classical single-player desktop GIS apps? E.g. I'm keeping an eye on the Zed code editor which had a vision of designing a code editor from the ground-up based on collaborative editing (see their intro blog post at https://zed.dev/blog/beta), and there are parallels to what GeoJupyter is trying to do here.

> ![An animation from the 1990s childrens' TV program "Captain Planet" depicting the planeteers combining their powers to summon... Captain Planet!](https://media1.tenor.com/m/hWtovfyLjNAAAAAd/captain-planet-let-our-powers-combine.gif)

Author: Too dorky? :laughing:
